### ISSUE TITLE
some issue in this project?

### ISSUE THREAD

**Issue by hongxiuzhe** [uber/AthenaX#5]
1.Is there some Class missed in the project on Github? Class "ExtendedJobDefinition, JobDefinition, JobDefinitionDesiredstate and JobDefinitionResource" are not in package com.uber.athenax.backend.api.

<img width="453" alt="2017-10-21 15 32 48" src="https://user-images.githubusercontent.com/18680152/31869826-1f8dfc5a-b7dd-11e7-8dac-264f4de0e752.png">

2. How do i sumbit SQL?
<img width="858" alt="2017-10-21 15 33 14" src="https://user-images.githubusercontent.com/18680152/31869834-29e430c0-b7dd-11e7-8e13-80fe5e4ffd4d.png">

@mranney @libber @haohui @djKooks @myhrvold

**Comment by haohui** [comment]
Thanks for trying out AthenaX. We usually use GitHub issues to track bugs -- you might get more responses by asking the questions on the mailing list.

1. The "missing" classes are generated during the compilation process. You can find them once you run `mvn clean compile install`
2. You can submit the query via the [REST](https://github.com/uber/AthenaX/blob/master/athenax-backend/src/main/resources/athenax-backend-api.yaml). Please see the [getting start guide]( http://athenax.readthedocs.io/en/latest/getting_started) for more details.

**Comment by hongxiuzhe** [comment]
It looks like bugs. I think an excellent open source framework needs full documentation for user. The documentation is useful for the development of frameworks. I think your doc is hard to understand and needs much time to run it.  I except a detailed and specific documents. @djKooks @haohui

**Comment by haohui** [comment]
AthenaX requires some level of development in order to accommodate the needs for different use cases. Please note that this is an intentional choice and the documentation state it explicitly.

If you find that the documentation is hard to follow please free free to open pull requests to improve it. Your contributions are appreciated.

**Comment by kination** [comment]
@hongxiuzhe @haohui I'm also a starter on this project so cannot help on this yet. I'll keep looking on source code and give a PR for document update.

**Comment by DoingDone9** [comment]
@hongxiuzhe  i submit job failed! Caused by: org.apache.flink.table.api.NoMatchedTableSourceConverterException: Could not find a TableSourceConverter for table type kafka+json.

**Comment by pzzs** [comment]
@hongxiuzhe i submit job failed! Do you know how  solve it? Thank you ! Caused by: org.apache.flink.table.api.NoMatchedTableSourceConverterException: Could not find a TableSourceConverter for table type kafka+json.
